pyrovider/meta/ioc.py

```diff
@@ -10,5 +11,12 @@ def get_obj(class_path: str) -> type:
         module_parts = class_path.split('.')
         module_name = ".".join(module_parts[:-1])
         module = importlib.import_module(module_name)
+        obj = module_parts[-1:][0]
 
-        return module.__dict__[module_parts[-1:][0]]
+        matches = re.findall(r'\["[\w-]+"]', obj)
```

Comment:
what is this for? I'm having troubling understanding the logic

Reply:
oh. is this to support dictionary indexing as part of the parameter, when referencing a config dict?

Reply:
Not sure if this is worth it. Wouldn't it be better to just convert the settings dict into a DictObject and use regular dot notation?

Reply:
yeah good idea. i can try that thx!